src/aggregate/expr/expression.c

```diff
@@ -245,19 +245,24 @@ static int evalOp(RSExprEvalCtx *ctx, RSExprOp *op, RSValue *result, char **err)
 }
 
 static int evalPredicate(RSExprEvalCtx *ctx, RSPredicate *pred, RSValue *result, char **err) {
-
+  int res;
   RSValue l = RSVALUE_STATIC, r = RSVALUE_STATIC;
   if (RSExpr_Eval(ctx, pred->left, &l, err) == EXPR_EVAL_ERR) {
     return EXPR_EVAL_ERR;
   }
+  if (pred->cond == RSCondition_And && !(res = RSValue_BoolTest(&l))) {
+    goto success;
```

Comment:
You did not set res value..

Reply:
@MeirShpilraien 
it is set in the line above.
Change to make cleaner